packages/react/src/combobox-popover-content/ComboboxPopoverContent.tsx

```diff
@@ -0,0 +1,23 @@
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+
+import { PopoverContent } from "../popover-content";
+import * as styles from "./ComboboxPopoverContent.css";
+
+type ComboboxPopoverContentProps = ComponentPropsWithoutRef<
+  typeof PopoverContent
+> &
+  styles.ContentVariants;
+
+export const ComboboxPopoverContent = forwardRef<
+  HTMLDivElement,
+  ComboboxPopoverContentProps
+>(({ className, minW = "trigger", ...props }, ref) => {
```

Comment:
no need to extract `minW` here since it'll get automatically overridden by `...props`
```suggestion
>((props, ref) => {
```

Reply:
Got it vai, I thought we needed to handle the `minW` explicitly, but as `ListboxBase` supports it, we can just forward the props to popover content.